cmd/mole/main.go

```
@@ -71,6 +146,39 @@ func startFromAlias(app cli.App) error {
 }
 
 func start(app cli.App) error {
+	if app.Detach {
+
+		cntxt := &daemon.Context{
+			PidFileName: "pid",
+			PidFilePerm: 0644,
+			LogFileName: "log",
+			LogFilePerm: 0640,
+			Umask:       027,
+			Args:        os.Args,
+		}
+
+		d, err := cntxt.Reborn()
+		if err != nil {
+			log.Fatal("Unable to run: ", err)
```

`log.Fatal` will cause `panic`. Please just return the error.